ambry-frontend/src/main/java/com.github.ambry.frontend/AccountAndContainerInjector.java

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
moved from `AmbryBlobStorageService` with two changes only

removed `RestUtils.Headers.AMBRY_CONTENT_TYPE` from `requiredAmbryHeadersForPutWithServiceId` and `requiredAmbryHeadersForPutWithAccountAndContainerName` because its presence is anyway verified in `RestUtils.buildBlobProperties()`